hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionUtils.java

```diff
@@ -619,4 +615,13 @@ static void updateStats(Optional<ServerStatisticTracker> optStats,
         metrics -> ResultStatsUtil.updateStats(metrics, serverName, regionName, regionLoadStats));
     });
   }
+
+  public static AsyncConnection createAsyncConnection(Configuration conf) throws IOException {
+    return FutureUtils.get(ConnectionFactory.createAsyncConnection(conf));
+  }
```

Comment:
What's the benefit of this, versus just doing `ConnectionFactory.createAsyncConnection(conf).get()` in ReplicationSink?

And do we need both of these methods, looks like just one is used?

Reply:
@bbeaudreault , thank you for review. Ok，I  think we could just use `ConnectionFactory.createAsyncConnection(conf).get() `in ReplicationSink， because seems there is no other code could benefit from this. I have updated the code.